src/database.ts

```diff
@@ -1234,7 +1233,7 @@ class Database extends GrpcServiceObject {
       typeof optionsOrCallback === 'object'
         ? (optionsOrCallback as GetSessionsOptions)
         : {gaxOptions: {}};
-    const gaxOpts: CallOptions = options.gaxOptions as CallOptions;
+    const gaxOpts: grpc.CallOptions = options.gaxOptions as grpc.CallOptions;
```

Comment:
No type definition or type cast is needed here as it is incorrect.
`gaxOptions` should be `google-gax.callOptions` as it will be correctly inferred from `options.gaxOptions` -> [`PagedRequest`](https://github.com/googleapis/nodejs-spanner/blob/12f4fd31242d25395b173a9aae2a21531978b18c/src/common.ts#L69) Not `grpc.callOptions`

Reply:
Thank you, fixed.